boxoffice/static/js/models/admin_order.js

```diff
@@ -0,0 +1,13 @@
+import {fetch, post} from './util.js';
+import {IndexModel} from './index.js';
+
+export const OrderModel = {
+  fetch: fetch,
+  post: post,
+  getUrl: function(ic_id, relativeUrl=false){
```

Comment:
1. The `Order` model will have to know about multiple URLs (to list orders, create an order, update or delete). So, this function should probably be designed with that in mind.
2. Is relativeUrl being used anywhere?